.github/workflows/reusable-e2e.yml

```diff
@@ -301,6 +301,265 @@ jobs:
             packages/e2e/cypress/screenshots/**
           retention-days: 5
 
+  check-changed-packages:
+    runs-on: ubuntu-latest
+    outputs:
+      has-changed-packages: ${{ steps.has-rn-changed-packages.outputs.has-changed-packages }}
+    steps:
+      - name: Checkout Amplify UI
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
+        with:
+          ref: ${{ inputs.commit }}
+          repository: ${{ inputs.repository }}
+          fetch-depth: 2
+          persist-credentials: false
+
+      - name: Determine if there were changes to react-native packages
+        id: has-rn-changed-packages
+        uses: ./.github/actions/changed-packages
+        with:
+          commit: ${{ inputs.commit }}
+          packages: |
+            packages/react-native
```

Comment:
Not related to this PR, but could we do something like this to skip tests on documentation only changes?